Source/Model/Mention.swift

```diff
@@ -36,7 +36,7 @@ struct Mention: Codable {
 //        self.type = nil
 //    }
     
-    init(link: Identifier, name: String? = nil , metadata: Blob.Metadata? = nil) {
+    init(link: Identifier, name: String? = nil, type: Int? = nil, metadata: Blob.Metadata? = nil) {
```

Comment:
please make this type an enum so that we document the possible values.

Reply:
sqlite doesn't support enum and i wasn't quite sure how to represent an enum in swift but have it be an Int in sqlite.

Reply:
you don't need sqlite's blessing for this. Just do something like this:

```swift
enum Example: Int {
   case fii = 1
   case faa = 2
   case foo = 3
 }
```

You might have to use `foo.rawValue` to please the compiler.